pages/DetailsTrip/[city]/index.js

```diff
@@ -93,60 +94,66 @@ export default function DetailsOfTrip() {
           ) : null}
         </StyledH4>
         <StyledH4>Hotels:</StyledH4>
-        <StyledUl>
-          {city.hotels.map((hotel) => (
-            <StyledList key={uuidv4()}>
-              <StyledSpan>{hotel.hotel}</StyledSpan>
-              {hotel.hotelPrice ? (
-                <StyledSpan>{hotel.hotelPrice} Euro</StyledSpan>
-              ) : null}
-            </StyledList>
-          ))}
+        <StyledBlox>
+          <StyledUl>
+            {city.hotels.map((hotel) => (
+              <StyledList key={uuidv4()}>
+                <StyledSpan>{hotel.hotel}</StyledSpan>
+                {hotel.hotelPrice ? (
+                  <StyledSpan>{hotel.hotelPrice} Euro</StyledSpan>
+                ) : null}
+              </StyledList>
+            ))}
 
+            <StyledDiv>
+              <StyledTotalPrice>Total Price:</StyledTotalPrice>
+              <StyledTotalPrice>{totalPriceOfHotels} Euro</StyledTotalPrice>
+            </StyledDiv>
+          </StyledUl>
+        </StyledBlox>
+        <StyledH4>Places to visit:</StyledH4>
+        <StyledBlox>
+          <StyledUl>
+            {city.places.map((place) => (
+              <StyledList key={uuidv4()}>
```

Comment:
If the place object has an id already on it, then i think it would be best to use that.

```suggestion
              <StyledList key={place.id}>
```